examples/npm-workspaces/client/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@examples-npm-workspaces/client",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "dev": "wait-port 3000 && tsx watch src/index.ts",
+    "start": "node dist/index.js"
+  },
+  "dependencies": {
+    "@trpc/client": "npm:@trpc/client@next",
+    "@examples-npm-workspaces/server": "*"
```

Comment:
For clarity this should be a dev dep cause we only need the type in dev. No runtime code is shared